src/components/MeasureResults/MeasureResultsTable.js

```diff
@@ -3,6 +3,7 @@ import { Divider, Grid, Typography } from '@mui/material';
 import React from 'react';
 import PropTypes from 'prop-types';
 import MeasureResultsRow from './MeasureResultsRow';
+import { IndexKind } from 'typescript';
```

Comment:
You don't use this. Please turn on and keep on your linter.

Reply:
This looks like an artifact from the PR that this was initially forked from on accident, and doesn't seem like it was called anywhere. Removing.